utils/time_test.go

```diff
@@ -15,36 +17,32 @@ func TestMillisFromTime(t *testing.T) {
 	actual := MillisFromTime(input)
 	expected := int64(1420115640000)
 
-	if actual != expected {
-		t.Fatalf("TestMillisFromTime failed, %v=%v", expected, actual)
-	}
+	assert.Equal(t, actual, expected)
 }
 
 func TestYesterday(t *testing.T) {
 	actual := Yesterday()
 	expected := time.Now().AddDate(0, 0, -1)
 
-	if actual.Year() != expected.Year() || actual.Day() != expected.Day() || actual.Month() != expected.Month() {
-		t.Fatalf("TestYesterday failed, %v=%v", expected, actual)
-	}
+	assert := assert.New(t)
+
+	assert.Equal(actual.Year(), expected.Year())
```

Comment:
For consistency with the rest of the tests, do you mind to remove the `assert := assert.New(t)` assignation, and pass the `t` parameter as first parameter to all the `assert.Equal` calls there?

Reply:
Ok